static/components/PoiCard.js

```diff
@@ -36,16 +36,22 @@ class PoiCard extends React.Component {
     return Math.trunc(R * c)
   }
 
-  hoverCard() {
-    this.base.classList.toggle('current-card')
-    const marker = document.getElementById(this.props.details._id)
-    marker.classList.toggle('selected')
-  }
+  // hoverCard() {
+  //   this.base.classList.toggle('current-card')
+  //   const marker = document.getElementById(this.props.details._id)
+  //   marker.classList.toggle('selected')
+  // }
 
   clickCard() {
-    highlight(this.props.details._id)
-    urlFor(this.props.details._id)
-    moveTo([this.props.details.position.longitude, this.props.details.position.latitude])
+    if (this.base.className == 'card current-card' && map.getZoom() < 14) {
```

Comment:
```
if(this.base.classList.contains('card') && this.base.classList.contains('current-card') &&
   map.getZoom…
```
- no space after `if`
- a className could be `current-card card` or `current-card card other-style` and it should still work.